src/containers/OversiktContainer.tsx

```diff
@@ -31,7 +31,10 @@ const OversiktContainer = ({ tabType }: Props): ReactElement => {
   }, [setTabType, tabType]);
 
   const ContainerContent = (): ReactElement => {
-    if (personoversiktQuery.isLoading || personregisterQuery.isLoading) {
+    if (
```

Comment:
Endret til isInitialLoading for queries som kan være disabled (Ref: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#disabled-queries https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading)